docs/awesome-babel/babel-plugin-add-module-exports.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: docs
```

Comment:
Also what would be interesting is instead of copying the readme we get the readme from npm?

https://unpkg.com/babel-plugin-add-module-exports@0.1.4/README.md - not sure why the readme wasn't published for https://unpkg.com/babel-plugin-add-module-exports/ for the latest version